stable_baselines3/common/callbacks.py

```diff
@@ -240,10 +240,31 @@ def _init_callback(self) -> None:
 
     def _on_step(self) -> bool:
         if self.n_calls % self.save_freq == 0:
-            path = os.path.join(self.save_path, f"{self.name_prefix}_{self.num_timesteps}_steps")
-            self.model.save(path)
+            model_path = os.path.join(self.save_path, f"{self.name_prefix}_{self.num_timesteps}_steps")
+            self.model.save(model_path)
             if self.verbose > 1:
-                print(f"Saving model checkpoint to {path}")
+                print(f"Saving model checkpoint to {model_path}")
+
+            if hasattr(self.model, "replay_buffer") and self.model.replay_buffer is not None:
```

Comment:
saving replay buffer and vec normalize stats should be opt-in (so deactivated by default).

Please also update the changelog and a test.

Reply:
Sure thing